### PR TITLE
Fixed #21046 -- Made startproject/app write platform-specific line endings.

### PR DESCRIPTION
**Pull request by claudep**

### CODE REVIEW

File: tests/admin_scripts/tests.py [hunk 1]
```diff
@@ -1899,6 +1899,11 @@ def test_simple_project(self):
         out, err = self.run_django_admin(args)
         self.assertNoOutput(err)
         self.assertTrue(os.path.isdir(testproject_dir))
+        # Ensure line endings are platform-specific
```
Comment (timgraham) [inline tests/admin_scripts/tests.py]: chop "ensure"

**Comment by timgraham** [comment]
Closing per ticket.

**Comment by claudep** [comment]
Even if the original issue is moot, what about keeping the patch (maybe without the test), as there is a small optimization (file copied instead of read/write)?

**Comment by timgraham** [comment]
Sure

**Comment by claudep** [comment]
@timgraham OK with #7416?

**Pull request closed without merging**